app/controllers/buyers/accounts/bulk/change_states_controller.rb

```diff
@@ -21,4 +22,9 @@ def create
     handle_errors
   end
 
+  private
+
+  def change_state_action_param
+    params.require(:change_states).require(:action)
```

Comment:
Do we want to fail hard if the `action` is not sent?
This will raise an `ActionController::ParameterMissing`, and  will display a generic page instead of the bulk action page with errors.

Reply:
I will keep the same behavior with a blank option, maybe we upgrade the whole bulk actions with Patternfly in the future.